### DISCUSSION TITLE
Older Releases

### DISCUSSION THREAD

**Discussion by coemgenuslcp** [pml-lang/pml-companion#61] [Ideas]
I would like to compare 2.1 with the older versions on my system, but I am too lazy to compile them and would rather get them from the Releases in GitHub, except that those are empty now. Is there any plan to make use of them?

**Comment by pml-lang** [comment]
A Github release for the latest version 2.1.0 has been [added](https://github.com/pml-lang/converter/releases/tag/v2.1.0).
Releases will also be created for future versions.

To download binary distribution files for older versions you can proceed like this:

Old Windows versions can be downloaded with a link like this one:
https://www.pml-lang.dev/downloads/windows/PML%20to%20HTML%20Converter-2.0.0.exe
The version number in the above link (2.0.0) can be replaced with any previously published version number.

For Linux distributions you can use a link like this for a deb file:
https://www.pml-lang.dev/downloads/linux/pml-to-html-converter_2.0.0-1_amd64.deb
... or like this for an rpm file:
https://www.pml-lang.dev/downloads/linux/pml_to_html_converter-2.0.0-1.x86_64.rpm
